### ISSUE TITLE
CellExecutionError from nbconvert package

### ISSUE THREAD

**Issue by appukuttan-shailesh** [scidash/sciunit#134]
'nbconvert' package seems to be undergoing restructuring, and this could require us to make some changes in sciunit.

The following would fail:
https://github.com/scidash/sciunit/blob/master/sciunit/utils.py#L27

I encountered the following when trying to build docs on "ReadTheDocs":
>  File "/home/docs/checkouts/readthedocs.org/user_builds/hippounit/envs/latest/lib/python3.7/site-packages/sciunit-0.2.2-py3.7.egg/sciunit/__init__.py", line 10, in <module>
    from .utils import settings, log
  File "/home/docs/checkouts/readthedocs.org/user_builds/hippounit/envs/latest/lib/python3.7/site-packages/sciunit-0.2.2-py3.7.egg/sciunit/utils.py", line 27, in <module>
    from nbconvert.preprocessors.execute import CellExecutionError
ImportError: cannot import name 'CellExecutionError' from 'nbconvert.preprocessors.execute' (/home/docs/checkouts/readthedocs.org/user_builds/hippounit/envs/latest/lib/python3.7/site-packages/nbconvert-6.0.0a1-py3.7.egg/nbconvert/preprocessors/execute.py)

It should probably be changed to:
`from nbconvert.preprocessors import CellExecutionError`

For now the changes are only in the [pre-release versions](https://pypi.org/project/nbconvert/#history), and the stable version (5.6.1) works fine.

**Comment by appukuttan-shailesh** [comment]
We should either go with:
```
try:
    from nbconvert.preprocessors.execute import CellExecutionError
except:
    from nbconvert.preprocessors import CellExecutionError
```
or change https://github.com/scidash/sciunit/blob/master/requirements.txt#L8
to:
`nbconvert<=5.6.1`

Could we make either of these changes and have a PyPI release? ReadTheDocs doesn't seem to have a workaround to prevent "pre-release" versions from being installed ([more here](https://github.com/readthedocs/readthedocs.org/issues/2339)).

**Comment by rgerkin** [comment]
@appukuttan-shailesh 
That try/except change sounds reasonable.  Do you want to open a PR for it?

**Comment by appukuttan-shailesh** [comment]
Sure. I have done this now via #135 .

**Comment by rgerkin** [comment]
@appukuttan-shailesh Merged and new pypi release 0.2.2.1 has been created: https://pypi.org/project/sciunit/

**Comment by appukuttan-shailesh** [comment]
This works now. Thanks :+1:
